### DISCUSSION TITLE
Performance advantage of using lanceDB for embedding and reranking instead of in app ?

### DISCUSSION THREAD

**Discussion by nicolaslecrique** [lancedb/lancedb#1925] [Q&A] [answered]
Hello,

Besides practicality, is there significant advantages (performance, other...) to use lanceDB to manage API calls to embedding and reranking models, instead of doing it in the application code (uncoupled from lanceDB) ?

Thanks!

**Comment by wjones127** [comment, marked as answer]
There isn't a performance advantage. It's mostly ease-of-use.
